frontend/src/RecordingPageComp/PlaybackContainer/PlaybackContainerComp.jsx

```diff
@@ -106,6 +99,10 @@ function PlaybackContainer({
     return pitches;
   }
 
+  // Takes in audio buffer data and performs an operation that compares two versions of the audio data
+  // Uses lag between wavelength data to determine when a vocal pitch is detected, if so this returns that frequency in hz
+  // Performs function at specific slice determined by the analyzePitchOverTime algorithim
+  // Inspired by Yin's mathematical algorithim for pitch detection
```

Comment:
Just a tip, you can use /* */ for multi-line comments

Reply:
Noted, will fix this from now on.